addons/mail/static/src/core/common/follower_list_view_model.js

```diff
@@ -0,0 +1,73 @@
+import { Record } from "@mail/core/common/record";
+import { rpc } from "@web/core/network/rpc";
+
+export class FollowerListView extends Record {
+    static id = "id";
+    /** @type {Object.<number, import("models").FollowerListView>} */
+    static records = {};
+    /** @returns {import("models").FollowerListView} */
+    static get(data) {
+        return super.get(data);
+    }
+    /**
+     * @param {Data} data
+     * @returns {import("models").FollowerListView|import("models").FollowerListView[]}
+     */
+    static insert(data) {
+        return super.insert(...arguments);
+    }
+
+    /** @type {number} */
+    id;
+    /** @type {number} */
+    thread_id;
+    /** @type {string} */
+    thread_model;
+
+    followers = Record.many("mail.followers");
+
+    /** @type {number} */
+    followersCount;
+
+    selfFollower = Record.one("mail.followers");
+
+    get followersFullyLoaded() {
+        return (
```

Comment:
Nitpick: ok for empty newlines between functions/methods, but attributes the empty newlines give too much room. It's easier to enumerate their listing when there's no empty newlines between them.